frontend/src/scss/navbar.scss

```diff
@@ -1,8 +1,12 @@
 .navbar{
     &__toolbar{
-        justify-content: flex-end;
+        justify-content: space-around;
```

Comment:
I think `justify-content: space-between;` might work a little bit better for this. What do you think?

Reply:
I think I had been going back-and-forth between the two when I originally made the changes, and can't quite remember why I chose space-around instead. Now that I look at it again, I would agree that space-between works better.